modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -737,3 +736,25 @@ export function getParticipantFromIndex(index: number): SignatureShareType {
       throw new Error(`Unknown participant index ${index}`);
   }
 }
+
+/**
+ * Helper function to verify wallet signatures for the bitgo-user and bitgo-backup shares.
+ * @param params
+ */
+export async function verifyWalletSignature(params: {
```

Comment:
does `verify wallet signatures` refer to verifying u-sigs? Please let's clarify in the comment